modules/xmpp/strophe.jingle.js

```diff
@@ -354,20 +373,24 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
             break;
         }
         case 'session-accept': {
+            logger.debug(`Received ${action} from ${fromJid}`, iq);
             this.eventEmitter.emit(
                 XMPPEvents.CALL_ACCEPTED, sess, $(iq).find('>jingle'));
             break;
         }
         case 'content-modify': {
+            logger.debug(`Received ${action} from ${fromJid}`, iq);
             sess.modifyContents($(iq).find('>jingle'));
             break;
         }
         case 'transport-info': {
+            logger.debug(`Received ${action} from ${fromJid}`, iq);
             this.eventEmitter.emit(
                 XMPPEvents.TRANSPORT_INFO, sess, $(iq).find('>jingle'));
             break;
         }
         case 'session-terminate': {
+            logger.debug(`Received ${action} from ${fromJid}`, iq);
```

Comment:
have to be careful with logging objects on mobile

Reply:
This has always been the case before, instead of logging every IQ, I am just logging the IQs for all messages except `session-initiate`, `source-add` and `source-remove`.

Reply:
Done.